Mathlib/Mathport/Rename.lean

```diff
@@ -54,11 +54,16 @@ def RenameMap.insert (m : RenameMap) (e : NameEntry) : RenameMap :=
 /-- Look up a lean 4 name from the lean 3 name. Also return the `dubious` error message. -/
 def RenameMap.find? (m : RenameMap) : Name → Option (String × Name) := m.toLean4.find?
 
+-- TODO: upstream into core/std
+instance [Inhabited α] : Inhabited (Thunk α) where
+  default := .pure default
+
 /-- This extension stores the lookup data generated from `#align` commands. -/
-initialize renameExtension : SimplePersistentEnvExtension NameEntry RenameMap ←
+-- wrap state in `Thunk` as we rarely actually query it
```

Comment:
I'm happy to merge this as is if we update this comment to 
```suggestion
-- wrap state in `Thunk` as downstream projects rarely actually query it; it only
-- gets queried when new `#align`s are added.
```

Reply:
That sounds reasonable. And we can decide whether to eliminate the duplication check in a future PR.